variables.ini.dist

```diff
@@ -37,6 +37,13 @@ ORG_NAME =
 SUPPORT_EMAIL =
 API_SERVER =
 BASE_URL =
+AUTH_USE_OVERRIDE =
+AUTH_ENABLE_MOCK =
+AUTH_ENABLE_MODEL =
+AUTH_ENABLE_GLOBUS =
+AUTH_ENABLE_OAUTH =
+AUTH_ENABLE_CAS =
+AUTH_ENABLE_LDAP =
```

Comment:
Okay - so I think we need to add a "todo" item to ensure this is populated in various 'build_envs'